### PR TITLE
Fix #3665: [RTL] High-fi Remove textdirection attribute from all the TextViews and fix text alignment

### LINKED ISSUE #3665: [RTL] High-fi Remove textdirection attribute from all the TextViews and fix textalignment issues
Remove the default direction of the text from everywhere in the project, that was added in order to support RTL. Fix textviews alignment for RTL wherever required.

### PR DESCRIPTION
**Pull request by veena14cs**
## Explanation
This PR fixes #3665. This PR removes `textdirection` property from Textviews used in the project and fixes text alignment for RTL.

## Screenshot LTR and RTL
Mobile
Screenshot_1628756888.... Screenshot_1628756836

Screenshot_1628756898......Screenshot_1628756844

Screenshot_1628804107....Screenshot_1628804064

Screenshot_1628804129.....Screenshot_1628804050

Tablet
Screenshot_1628799081.....Screenshot_1628797646

Screenshot_1628799088.....Screenshot_1628797638

Screenshot_1628799046....Screenshot_1628798965

Screenshot_1628799038.......Screenshot_1628798972

Screenshot_1628801987.....Screenshot_1628801720

Screenshot_1628801975....Screenshot_1628801736

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by rt4914 — CHANGES_REQUESTED** [review]
@veena14cs Please add RTL screenshots for all files which have been changed.

**Comment by veena14cs** [comment]
> @veena14cs Please add RTL screenshots for all files which have been changed.

@rt4914 Added screenshots. PTAL

File: app/src/main/res/layout-land/topic_info_fragment.xml [hunk 1]
```diff
@@ -26,58 +26,58 @@
...
         android:fontFamily="sans-serif"
         android:text="@{viewModel.topic.name}"
         android:textColor="@color/oppiaPrimaryText"
-        android:textDirection="locale"
         android:textSize="24sp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"/>
```
Comment (rt4914) [inline app/src/main/res/layout-land/topic_info_fragment.xml]: Nit: space before `/>`

Comment (rt4914) [reply app/src/main/res/layout-land/topic_info_fragment.xml]: Basically use reformatting shortcut in all files before merging.

**Review by rt4914 — APPROVED** [review]
Approved with one nit change.

File: app/src/main/res/layout-land/topic_info_fragment.xml (hunk 1 above)
Comment (veena14cs) [reply app/src/main/res/layout-land/topic_info_fragment.xml]: Thanks @rt4914. Done reformatting here and elsewhere.

**Pull request merged**